airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -270,6 +271,14 @@
                 "order": 5,
                 "min": 120,
                 "max": 1200
+              },
+              "lsn_commit_behaviour": {
```

Comment:
Could you post a quick screenshot of how this looks like?

Reply:
<img width="648" alt="Screenshot 2023-01-03 at 1 11 47 PM" src="https://user-images.githubusercontent.com/28527634/210316505-663eca58-fef4-4786-814a-5c0c920a5931.png">